sentry-android-core/src/main/java/io/sentry/android/core/ActivityLifecycleIntegration.java

```diff
@@ -328,26 +336,16 @@ public synchronized void onActivityStarted(final @NotNull Activity activity) {
   @SuppressLint("NewApi")
   @Override
   public synchronized void onActivityResumed(final @NotNull Activity activity) {
-    if (!firstActivityResumed) {
-
-      // we only finish the app start if the process is of foregroundImportance
-      if (foregroundImportance) {
-        // sets App start as finished when the very first activity calls onResume
-        AppStartState.getInstance().setAppStartEnd();
-      } else {
-        if (options != null) {
-          options
-              .getLogger()
-              .log(
-                  SentryLevel.DEBUG,
-                  "App Start won't be reported because Process wasn't of foregroundImportance.");
-        }
-      }
+    // finishes app start span
+    @Nullable final SentryDate appStartEndTime = AppStartState.getInstance().getAppStartEndTime();
+    if (appStartSpan != null
```

Comment:
`h`: actually, I just realized - if the users have removed/disabled the `SentryPerformanceProvider` via manifest, we'd never finish the app start span, because the `appStartEndTime` is not gonna be ever set. If you look at `SentryAndroid` - we set the app start times there to mitigate the removal of the provider. So I believe we should also have a fallback here in case `appStartEndTime` is not available

Reply:
yea, but that's what I'm saying - in [SentryAndroid](https://github.com/getsentry/sentry-java/blob/f6a135d1ef81f9cd70d72608e8beb8c2103f69bd/sentry-android-core/src/main/java/io/sentry/android/core/SentryAndroid.java#L84-L86) we actually set the app start times again for the case when the provider is disabled, so it will be set still :)